### PR TITLE
Update Altium Docs

### LINKED ISSUE #7: Add Instructions for using QFSAE Altium Library to README

### PR DESCRIPTION
**Pull request by llim1**
More detailed additions to altium docs in preparation for the tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,14 +20,55 @@ The QFSAE team designs its boards using Altium. If you do not have Altium Design
...
+
+Please try to release your license whenever you are not working in Altium we don't run out of seats! 
+
+*Note:* If you are having issues using a license, verify your system time is correct. This issue is common on Windows/Linux dual boots where rebooting back into Windows can offset the time. 
+
+### Part Library Setup
+For the majority of projects the only library required is the JLCPCB Altium component library. It can be located in the [libraries](https://github.com/qfsae/pcb/tree/master/libraries) folder of this repository, under **"JLCSMT_LIB"**. 
+
+#### Adding Parts Libraries
+The JLC library is a linked repository, so start by navigating to the [libraries](https://github.com/qfsae/pcb/tree/master/libraries) folder of this repository, clicking through to the JLC library repository and downloading it. Save the library in a safe directory (ie. not Downloads!) We will be adding it to your Altium installation so it is available for use in all projects.
```
Comment (ethanmpeterson) [inline README.md]: No need to have people explicitly download the linked library repository as we can manage it through sub modules. People will have the library downloaded by running `git submodule update --init` after they clone this repository. The same applies for Celestial library files as well.

File: README.md [hunk 2]
```diff
@@ -14,14 +20,55 @@ The QFSAE team designs its boards using Altium. If you do not have Altium Design
...
 
 ![Altium Panel](https://i.imgur.com/d9237hu.png)
 
-For a broader selection of high quality footprints please follow the installation guide for the [Celestial Altium Library](https://altiumlibrary.com/GetStarted). The component you are looking for will not always appear in the JLC PCB or Celestial libraries. If this is the case, then use Altium's manufacturer part search feature.
+For more information about available part libraries, see the [documentation](https://www.altium.com/documentation/altium-designer/integratedlibrary-dlg-addremovelibrariesformavailable-file-based-libraries-ad). 
+
+#### Other Libraries
+For a broader selection of high quality footprints we use the open source Celestial Library. To install this library, follow the installation guide for the [Celestial Altium Library](https://altiumlibrary.com/GetStarted). 
+
+The component you are looking for will not always appear in the JLC PCB or Celestial libraries. If this is the case, then use Altium's manufacturer part search feature. Try to avoid using online part footprints wherever possible, as it can become difficult to maintain.
```
Comment (ethanmpeterson) [inline README.md]: Question: Should we mention that if you cannot find a footprint anywhere for a specialized part that there is the option of doing a custom footprint and throwing it in a QFSAE library?

File: README.md [hunk 3]
```diff
@@ -82,9 +129,10 @@ Altium has integrated Git support. The instructions below show how to configure
 
 To contribute a new PCB design, follow these steps:
 
+- Make sure you are up to date on master, then start a new branch using `git branch -b branchName`.
```
Comment (ethanmpeterson) [inline README.md]: get up to date on master using `git pull`

**Review by ethanmpeterson — COMMENTED** [review]
Looks good to me! Left a couple questions / comments.

File: README.md (hunk 2 above)
Comment (llim1) [reply README.md]: I'll add a blurb about how we may start an internal library but have not yet

**Comment by llim1** [comment]
I think this is a good amount of detail for installation and set up, at least for tutorial 1.

**Review by ethanmpeterson — APPROVED** [review]
LGTM

**Pull request merged**